docs/content/Aggregations.md

```diff
@@ -36,20 +36,36 @@ Computes the sum of values as 64-bit floating point value. Similar to `longSum`
 
 ### Min / Max aggregators
 
-#### `min` aggregator
+#### `doubleMin` aggregator
 
-`min` computes the minimum metric value
+`doubleMin` computes the minimum among metric value from rows and Double.POSITIVE_INIFINITY
 
 ```json
-{ "type" : "min", "name" : <output_name>, "fieldName" : <metric_name> }
+{ "type" : "doubleMin", "name" : <output_name>, "fieldName" : <metric_name> }
 ```
 
-#### `max` aggregator
+#### `doubleMax` aggregator
 
-`max` computes the maximum metric value
+`doubleMax` computes the maximum amont metric value from rows and Double.NEGATIVE_INIFINITY
 
 ```json
-{ "type" : "max", "name" : <output_name>, "fieldName" : <metric_name> }
+{ "type" : "doubleMax", "name" : <output_name>, "fieldName" : <metric_name> }
+```
+
+#### `longMin` aggregator
+
+`longMin` computes the minimum among metric value from rows and Long.MAX_VALUE
+
+```json
+{ "type" : "longMin", "name" : <output_name>, "fieldName" : <metric_name> }
+```
+
+#### `longMax` aggregator
+
+`longMax` computes the maximum amont metric value from rows and Long.MIN_VALUE
```

Comment:
typo, not sure "among" is grammatically correct here, maybe "computes the maximum of all metric values and `Long.MIN_VALUE`" is clearer?